htdocs/fourn/class/fournisseur.facture.class.php

```diff
@@ -229,6 +229,7 @@ class FactureFournisseur extends CommonInvoice
 
 	public $mode_reglement_id;
 	public $mode_reglement_code;
+	public $special_code;
```

Comment:
There is no column special code on supplier invoice. So we should not have this property.